Checkbox/Checkbox.jsx

```diff
@@ -13,6 +13,9 @@ export default class Checkbox extends MaterialComponent {
   componentDidMount() {
     this.MDComponent = new MDCCheckbox(this.control);
   }
+  componentWillUnmount() {
+    this.MDComponent.destroy();
```

Comment:
Ah ! I was not aware ```destroy``` wasn't part of the public API : you're absolutely right.
I'll safe guard every call to this method.